xapian-core/tests/api_closedb.cc

```diff
@@ -49,7 +49,6 @@ struct closedb1_iterators {
     Xapian::Database db;
     Xapian::Document doc1;
     Xapian::PostingIterator pl1;
-    Xapian::PostingIterator pl2;
```

Comment:
I couldn't see why pl2 is present. So removed. Let me know if it's there for some purpose, I will undo this change.

Reply:
Good spot.  The history isn't very informative (it was added like this) but I think the idea was to test an iterator that had been advanced before the database was closed (as well as `pl1` which hasn't been).  That seems useful as it might affect the behaviour, but to be useful we need to actually check `pl2` in similar ways to `pl1` below (just with different values in the checks because it is one further along).